simp_le.py

```diff
@@ -372,10 +371,11 @@ def get_output_or_fail(self, command):
     def persisted(self):
         """Call the external script and see which data is persisted."""
         output = self.get_output_or_fail('persisted')
+        persisted_keys = output.split('\n', 1)[0].split()
```

Comment:
I'll revert for now, backwards and forward compatibility is a bit hard to reach without the command and client being able to advertise the supported interface version.